ReSharper.FSharp/src/FSharp.Psi.Intentions/src/Intentions/TupleAnnotationAction.fs

```diff
@@ -0,0 +1,26 @@
+namespace JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Intentions
+
+open JetBrains.ReSharper.Feature.Services.ContextActions
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Tree
+open JetBrains.ReSharper.Psi.ExtensionsAPI
+open JetBrains.ReSharper.Resources.Shell
+
+[<ContextAction(Name = "AnnotateBinding", Group = "F#",
+                Description = "Annotate binding with explicit type")>]
+type TupleAnnotationAction(dataProvider: FSharpContextActionDataProvider) =
```

Comment:
It seems there should be no separate action for a specific pattern kind. I think there should be an action for a simple reference pattern and it should probably also provided a scoped action for the whole containing compound pattern to cover cases like this.